go/oasis-node/cmd/node/node_control.go

```diff
@@ -359,7 +360,12 @@ func (n *Node) getBundleStatus() ([]control.BundleStatus, error) {
 	bundles := make([]control.BundleStatus, 0, len(manifests))
 
 	for _, manifest := range manifests {
-		explodedComponents, err := bundleRegistry.GetComponents(manifest.ID, manifest.GetVersion())
+		ronl := manifest.GetComponentByID(component.ID_RONL)
+		if ronl == nil {
+			return nil, fmt.Errorf("manifest has no RONL component")
```

Comment:
Ideally these errors should not cause the entire node status to become unavailable.

Reply:
This error is not possible as the registry only returns RONL manifests. But I can skip non-ronl manifests, removing the error.